src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -11302,20 +11302,227 @@ await using (new S2())
     }
 }
 ";
-            var comp = CreateCompilation(src, targetFramework: s_targetFrameworkSupportingByRefLikeGenerics, options: TestOptions.ReleaseExe);
+            var comp1 = CreateCompilation(src1, targetFramework: s_targetFrameworkSupportingByRefLikeGenerics, options: TestOptions.ReleaseExe);
 
-            comp.VerifyDiagnostics(
-                // PROTOTYPE(RefStructInterfaces): follow up on recent changes due to https://github.com/dotnet/csharplang/blob/main/proposals/ref-unsafe-in-iterators-async.md
-                // (18,22): error CS9104: A using statement resource of type 'S2' cannot be used in async methods or async lambda expressions.
```

Comment:
Consider keeping this verification for LangVersion=12

Reply:
> Consider keeping this verification for LangVersion=12

I do not find that particularly interesting for the scenarios tested in this PR because the ref-struct-interfaces feature isn't supported in C# 12 either.